### ISSUE TITLE
Instalar

### ISSUE THREAD

**Issue by betimmm** [DarkSupremo/BJShare_SickRage_Provider#1]
Bom dia amigo, teria como explicar melhor como instalar o provedor BJ? Não sou da área e não entendi muito bem a explicação da página inicial? É algo via comando por exemplo ou faço pela interface web do sickrage?

**Comment by uilton-oliveira** [comment]
**1º Passo**: Faça o download do [BJ_Sickrage](https://github.com/DarkSupremo/BJShare_SickRage_Provider/archive/master.zip), extraia em alguma pasta e abra ela pelo Terminal / Prompt de Comando

**2º Passo**: Adicione o provider do BJ no seu SickRage. Para isso, é preciso executar o script *install.py* no seu Terminal/Prompt de Comando

Abra o Terminal/Command Line e execute o seguinte comando: **python install.py path_do_sickrage**, onde local_do_sickrage é o caminho para onde você mantém a instalação do SickRage.

Caso ocorra tudo certo, Prossiga para o 3º Passo.


Caso algum erro ocorra, é possível fazer manualmente.

Para isso, copie o arquivo **bjshare.py** para **path_do_sickrage/sickbeard/providers**

Depois copie o logo **bj_share.png** para **path_do_sickrage/gui/slick/images/providers**

Por último, vá até o arquivo **path_do_sickrage/sickbeard/providers/__init__.py**.
Na linha *25* adicione **bjshare** após **from sickbeard.providers import**
Na linha *32* adicione **'bjshare',** (sim, com aspas e a vírgula após fechar as aspas) no início da linha.

![](http://i.imgur.com/4vLsdHI.png)

**3º Passo**: Reinicie o SickRage.

**4º Passo**: Setup do BJ no SickRage.

Em **Config > Search Providers**, na aba *Provider Properties*, selecione o BJ.

![](http://i.imgur.com/RYXSSnX.png)

Na aba *Provider Options* selecione o BJ-Share em **"Configure provider"**, em **"Cookies"**, coloque: **session=#SESSION_COOKIE#**

**OBS**: Substituir **#SESSION_COOKIE#** pelo seu cookie de sessão, basta usar o **DevTools** do chrome (aperte F12), vá na aba **Applications** -> no canto esquerdo, expanda **Cookies** e clique no **"bj-share.me"** e copie o valor do **"session"**

**5º Passo**: Salve suas configurações do SickRage e voilà! O SickRage terá acesso ao BJ para baixar suas séries.

OBS 01: Como as séries aqui no BJ acompanham legendas, aconselho que deixe o SickRage copiar arquivos anexos. Para isso, vá em **Config > Post-Processing**, desmarque **Delete associated files** e em **Keep associated file extensions** veja se a extensão **srt** está inclusa.

![](http://i.imgur.com/zGmmZNx.png)

OBS 02: Garanta que a opção **Agrupamento dos torrents** está **ativada** em suas Configurações aqui no site do BJ.

**Comment by betimmm** [comment]
Amigo, desde já agradeço pelo guia detalhado.

Baixei o putty e consegui conectar, contudo, não encontro essa pasta "providers". Procurei pelo windows explorer também (via samba).

O máximo que consegui encontrar, relacionado ao sickrage, foi a pasta "\Userdata\addon_data\service.sickrage" mas dentro dela não tem nenhuma "providers", apenas as pastas "backup", "cache" e "logs" e arquivos.

Fiz algo de errado?

**Comment by uilton-oliveira** [comment]
Esta pasta não é a de instalação do sickrage.. tem que ver onde está os arquivos de instalação, o conteúdo será esses:

![image](https://user-images.githubusercontent.com/7059651/29252958-cf7509ba-8047-11e7-8273-6d3da8ef5c04.png)

Tente buscar por algum desses arquivos, por exemplo: find -name sickbeard.db

**Comment by betimmm** [comment]
Consegui acessar as pastas! Uma dúvida, onde eu deixo os arquivos do BJ_sickrage? Descompactei na área de trabalho. Tenho que mover para o libreelec para executar o comando de instalação?

**Comment by uilton-oliveira** [comment]
Pode ser em qualquer lugar, o script de instalação vai por os arquivos
necessários na pasta do sickrage, basta colocar o local correto de
instalação do sickrage na hora de executar

Em 13 de ago de 2017 6:19 PM, "betimmm" <notifications@github.com> escreveu:

**Comment by betimmm** [comment]
Amigo, quando dou o comando de instalar, aparece depois alguma mensagem dizendo que deu certo? Fiz aqui mas não recebi qualquer mensagem (de erro ou de sucesso) e não apareceu o bj na lista de provider, provavelmente terei que fazer o processo manual?

Fiz o seguinte, descompactei a pasta do seu link no meu HD externo que está acoplado ao odroid com libreelec. Entrei nessa pasta e executei o comando de instalar. Fiz algo errado?

**Comment by uilton-oliveira** [comment]
Sim, tem uma mensagem falando que foi copiado com sucesso... tente fazer o método manual então, é bem tranquilo de fazer

**Comment by betimmm** [comment]
Beleza! Essa semana eu tento fazer o manual e te dou um feedback por aqui, obrigado pelo esforço na ajuda!

**Comment by betimmm** [comment]
Amigo, consegui! Usei o winscp que tem interface gráfica, o que facilitou e MUITO tendo em vista que não sou familiarizado com os comandos do SSH. O bjshare já apareceu nos provedores!

Contudo, tentei procurar por episódio de séries que sei que tem no site, contudo, ele sempre dá erro e diz que não encontrou (2017-08-15 20:29:39 INFO SEARCHQUEUE-MANUAL-121361 :: Unable to find a download for:....).

É alguma configuração que fiz de errado?

**Comment by uilton-oliveira** [comment]
Ainda não testei pra series, só anime, mas vou dar uma testada, amanhã te
respondo

Em 15 de ago de 2017 8:31 PM, "betimmm" <notifications@github.com> escreveu:

**Comment by betimmm** [comment]
Valeu amigo, fico no aguardo.
